server/js/sockets.js

```diff
@@ -299,6 +299,14 @@ sockets.init = function socketInit(server) {
     socket.on('stop-camera-recording', () => {
       mqttClient.publish(Camera.recording_stop);
     });
+
+    socket.on('start-das-recording', () => {
+      mqttClient.publish(WirelessModule.all().start);
+    });
+
+    socket.on('stop-das-recording', () => {
+      mqttClient.publish(WirelessModule.all().stop);
+    });
```

Comment:
Unfortunately we can't publish to wildcard topics, so we'll have to publish to each module separately :/

Reply:
done
d1e47ef